File: tests/fixtures/openai-realtime/openai-realtime-avoid-dated-preview-snapshots-broken/02-inline-literal-url.ts
import WebSocket from 'ws';

class AudioInterceptor {
  setup(apiKey: string): WebSocket {
    return new WebSocket(
      'wss://api.openai.com/v1/realtime?model=gpt-4o-realtime-preview-2025-06-03',
      { headers: { Authorization: `Bearer ${apiKey}` } },
    );
  }
}

export default AudioInterceptor;

File: tests/fixtures/openai-realtime/openai-realtime-avoid-dated-preview-snapshots-broken/02_inline_literal_url.py
import websockets


class AudioInterceptor:
    def setup(self, api_key):
        return websockets.connect(
            "wss://api.openai.com/v1/realtime?model=gpt-4o-realtime-preview-2025-06-03",
            additional_headers={"Authorization": f"Bearer {api_key}"},
        )
